samples/net/gptp/src/gptp.c

```diff
@@ -75,9 +84,19 @@ void init_testing(void)
 {
 	uint32_t uptime = k_uptime_get_32();
 	int ret;
+	const struct device *clk;
+
+	clk = device_get_binding("PTP_CLOCK");
+	if (clk != NULL) {
+		k_object_access_grant(clk, k_current_get());
+	}
 
 	if (run_duration == 0) {
-		return;
+		/* USER BEGIN MAIN.C*/
+		while (1) {
+
+			k_msleep(200); /* sleep time in ms */
+		}
```

Comment:
Why do we need this loop?

Reply:
@hakehuang , would you like to comment on this?

Reply:
I have just reviewed this, and I can answer this myself. It was my bad. That code snippet was there as part of a modification that I decided should not be part of this pull request. I removed the critical stuff, but I mistakenly left this there.

I have now removed it, so I'll mark this conversation as resolved. Thank you for pointing it out.